{{cookiecutter.project_name}}/.github/workflows/main.yml

```diff
@@ -29,7 +29,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.9", "3.10", "3.11", "3.12", "3.13"]
+        python-version: {{ cookiecutter._python_version_matrix }}
```

Comment:
I just tried it by selecting main_python_version 3.13, and I ended up with only 3.13?  IMO we should still test for all current Python versions that are not EOL.